etna/search/forms.py

```diff
@@ -75,41 +106,38 @@ class BaseCollectionSearchForm(forms.Form):
         required=False,
         widget=forms.TextInput(attrs={"class": "search-filters__search"}),
     )
-    levels = DynamicMultipleChoiceField(
+    level = DynamicMultipleChoiceField(
         label="Level",
-        filter_key="level",
+        choices=LEVEL_CHOICES,
```

Comment:
Thanks @TNA-Allan:

1. Like all of the other choice fields here, these fields have their choices values replaced too... but that is AFTER the field has been used for validation. The only difference is that the label values from the original choices are reused, rather than being forgotten about completely
2. True, but that is literally what was specified in the ticket requirements. Have previously discussed with @sdwilkes, and these value do not change often - it's a known/deliberate drawback.